docs/how-to-guides/advanced-user-guides/enable-api-audit-log-in-downstream-clusters.md

```diff
@@ -77,7 +104,88 @@ spec:
             rke.cattle.io/control-plane-role: 'true'
 ```
 
-For more information about cluster configuration, refer to the REK2 or K3s cluster configuration reference pages.
+:::tip
+
+Alternatively, you can use the directive `machineSelectorConfig` with proper machineLabelSelectors to achieve the same effect.
+
+:::
+
+For more information about cluster configuration, refer to the RKE2 cluster configuration reference pages.
+
+</TabItem>
+
+<TabItem value="K3s" default>
+
+:::note
+
+This feature is available in Rancher v2.7.2 and above.
+
+:::
+
+You can utilize the `machineSelectorFiles` to deliver the audit policy file to the control plane nodes, and the `machineGlobalConfig` to set the options on kube-apiserver.
+
+As a prerequisite, you need to create a secret or configmap which will be the source of the audit policy.
+
+The secret or configmap must meet the following two requirements:
+
+1. It must be in the `fleet-default` namespace where the Cluster object exists.
+2. It must have the annotation `rke.cattle.io/object-authorized-for-clusters: cluster-name1,cluster-name2` which permits the target clusters to use it.
```

Comment:
Adding suggestion to follow line 166 where the placeholder is enclosed within `<>`, e.g. `<customized-path>` so it's explicit that it's a user provided value.

```suggestion
2. It must have the annotation `rke.cattle.io/object-authorized-for-clusters: <cluster-name1>,<cluster-name2>` which permits the target clusters to use it.
```